jenkins/Pipeline.gy

```diff
@@ -28,6 +28,12 @@ pipeline {
                 sh 'jenkins/run tox -e agent-py36 -- agent'
             }
         }
+        stage('Server Alembic Migrations Check') {
+            steps {
+                echo 'Verify alembic migrations cover latest database schema'
+                sh 'jenkins/run-alembic-migrations-check'
+            }
+        }
```

Comment:
I _think_ this would be better if we could package it as a test instead of having it as a separate pipeline stage.

Reply:
Per our earlier discussion, `jenkins/run` is set up so that it can be invoked inside a container which was run via `jenkins/run`.  So, the invocation of `jenkins/run` inside `jenkins/run-alembic-migrations-check` should be fine, but the script is also using `podman` directly, wherein lies the (first) problem.  I tried out the following change
```
diff --git a/jenkins/run-alembic-migrations-check b/jenkins/run-alembic-migrations-check
index 67b713730..a2cba893c 100755
--- a/jenkins/run-alembic-migrations-check
+++ b/jenkins/run-alembic-migrations-check
@@ -1,6 +1,7 @@
 #!/bin/bash -e
 
-podman run --name postgresql-alembic \
+PODMAN=$([ -n "${CONTAINER_HOST}" ] && echo podman-remote || echo podman)
+${PODMAN} run --name postgresql-alembic \
     --detach \
     --rm \
     --network host \
```
which allowed me to run the script via `jenkins/run`, and the PostgreSQL container seems to start up OK.  (I guess we should find a more general way to encapsulate the conditional definition for `${PODMAN}`.)

However, this brought me to the second problem:  inside the CI container (which is what I had `jenkins/run` use), there is no `nc` command.  But, this wouldn't be too hard to address, either....

So, we _should_ be able to package this as a test, we just need to enhance the infrastructure a little.

Reply:
Seems like a good change for a follow-up PR since this means rebuilding container images and verifying a larger impact does not ensue.

Reply:
Tracked in issue #3185.